src/components/applicationDetailsContainer.tsx

```diff
@@ -0,0 +1,27 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+function ApplicationDetailsContainer() {
+  return (
+    <div className="flex flex-col gap-y-8 ">
```

Comment:
Actually for consistency, you can use the same [sidebar](https://github.com/GenomicDataInfrastructure/gdi-userportal-frontend/blob/main/src/app/datasets/%5Bid%5D/Sidebar.tsx) component as we have in dataset page. We could think of extracting it out as generic component where content can be passed as child